ckanext/canada/commands.py

```diff
@@ -408,6 +434,35 @@ def copy_datasets(self, remote, package_ids=None):
             else:
                 action = 'updated'
                 portal.action.package_update(**source_pkg)
+                # create datastore table and views for each resource
+                for src_res in source_pkg['resources']:
+                    res_id = src_res['id']
+                    if res_id in source_pkg.keys():
+                        # delete resource and recreate table to capture changes made to columns or data
+                        # it would also avoid appending records to an existing resource table by datastore_create
+                        try:
+                            portal.call_action('datastore_delete', {"id": res_id, "force": True})
```

Comment:
can't you check the resource hash to prevent deleting and reloading data every time there's any change to a dataset? the same dataset appears many times in the activity logs as people are making changes so avoiding extra work here is important